www/configurator/src/index.js

```diff
@@ -33,36 +33,63 @@ if (process.env.NODE_ENV === 'production') {
   cssPath = 'test-dist/uncompiled-css/templates/';
 }
 
+// Function to get params on the page
+function getUrlParams() {
```

Comment:
Was planning on doing this in another PR too keep the CL size low, but I can definitely do it here :)